packages/shared/src/contexts/BootProvider.tsx

```diff
@@ -202,50 +199,51 @@ export const BootDataProvider = ({
       }
 
       const updated = updateLocalBootData(cachedData, updatedData);
-
-      queryClient.setQueryData<Partial<Boot>>(BOOT_QUERY_KEY, (previous) => {
-        if (!previous) {
-          return updated;
-        }
-
-        return { ...previous, ...updated };
-      });
+      setCachedBootData(updated);
     },
-    [queryClient],
+    [],
   );
 
   const updateUser = useCallback(
     async (newUser: LoggedUser | AnonymousUser) => {
-      updateQueryCache({ user: newUser });
+      updateBootData({ user: newUser });
       await queryClient.invalidateQueries({
         queryKey: generateQueryKey(RequestKey.Profile, newUser),
       });
     },
-    [updateQueryCache, queryClient],
+    [updateBootData, queryClient],
   );
 
   const updateSettings = useCallback(
-    (updatedSettings) => updateQueryCache({ settings: updatedSettings }),
-    [updateQueryCache],
+    (updatedSettings) => updateBootData({ settings: updatedSettings }),
+    [updateBootData],
   );
 
   const updateAlerts = useCallback(
-    (updatedAlerts) => updateQueryCache({ alerts: updatedAlerts }),
-    [updateQueryCache],
+    (updatedAlerts) => updateBootData({ alerts: updatedAlerts }),
+    [updateBootData],
   );
 
   const updateExperimentation = useCallback(
     (exp: BootCacheData['exp']) => {
-      updateLocalBootData(bootData, { exp });
+      updateLocalBootData(cachedBootData, { exp });
     },
-    [bootData],
+    [cachedBootData],
   );
 
   gqlClient.setHeader(
     'content-language',
     (user as Partial<LoggedUser>)?.language || ContentLanguage.English,
   );
 
+  useEffect(() => {
+    if (remoteData) {
+      setInitialLoad(initialLoad === null);
+      updateBootData(remoteData);
+    }
```

Comment:
This missing? https://github.com/dailydotdev/apps/commit/80281fe95eafaefa87846dae7bff3f7e12037291#diff-2279addce4c236245d34c6431e5269d47f22fd68c58f83c257dafa1b4e1b4599L197

Reply:
We don't need that anymore. That's only for anonymous users changing their settings. They don't have access to that anymore.